### PR TITLE
ACLs: Remove server usage

### LINKED ISSUE #8125: Disconnect ACL server from lakeFS

### PR DESCRIPTION
**Pull request by N-o-Z**
Closes #8125

Changes:
- UI
- Documentation
- Removal of development params

### CODE REVIEW

**Comment by N-o-Z** [comment]
@ozkatz, @talSofer 
Please review documentation and UI sections

File: docs/security/access-control-lists.md [hunk 1]
```diff
@@ -10,67 +10,61 @@ redirect_from:
...
-```
-lakefs migrate up
+ACL server was moved out of core lakeFS and into a new package under `contrib/auth/acl`.
+Though we [decided](https://lakefs.io/blog/why-moving-acls-out-of-core-lakefs/) to move ACLs out, we are committed to making sure existing users who still need the use of ACLs can continue using
+this feature.
+In order to do that, users will need to run the separate ACL server as part of their lakeFS deployment environment and configure lakeFS to work with it.
+
+### ACL server Configuration
+
+Under the `contrib/auth/acl` you will be able to find an ACL server reference.
```
Comment (ozkatz) [inline docs/security/access-control-lists.md:37]:
```suggestion
Under the `contrib/auth/acl` you will be able to find an ACL server reference.

{: .note .warning}
> This implementation is a reference and is not fit for production use. 
> 
> For a more robust authorization solution, please see [Role-Based Access Control](./rbac.html), available in [lakeFS Cloud]({% link cloud/index.md %}) and [lakeFS Enterprise]({% link enterprise/index.md %}). 


```

**Review by ozkatz — APPROVED** [review]
Thanks!

**Review by guy-har — APPROVED** [review]
Looks Good!

**Pull request merged**